Tasks/DockerV2/dockerpush.ts

```diff
@@ -103,17 +108,19 @@ export function run(connection: ContainerConnection, outputUpdate: (data: string
     return promise;
 }
 
-async function publishToImageMetadataStore(connection: ContainerConnection, imageName: string, tags: string[], digest: string, dockerFilePath: string): Promise<any> {
+async function publishToImageMetadataStore(connection: ContainerConnection, imageName: string, tags: string[], digest: string, dockerFilePath: string, imageSize: string): Promise<any> {
     // Getting imageDetails
     const imageUri = getResourceName(imageName, digest);
     const baseImageName = getBaseImageNameFromDockerFile(dockerFilePath);
-     const layers = await dockerCommandUtils.getLayers(connection, imageName);
+    const layers = await dockerCommandUtils.getLayers(connection, imageName);
 
     // Getting pipeline variables
-    const buildId = parseInt(tl.getVariable("Build.BuildId"));
-    const buildDefinitionName = tl.getVariable("Build.DefinitionName");
-    const buildVersion = tl.getVariable("Build.BuildNumber");
-    const buildDefinitionId = tl.getVariable("System.DefinitionId");
+    const hostType = tl.getVariable("System.HostType").toLowerCase();
+    const runId = hostType === "build" ? parseInt(tl.getVariable("Build.BuildId")) : parseInt(tl.getVariable("Release.ReleaseId"));
```

Comment:
build can be const and also it can be read from a common variable.

Reply:
Made the string build a const. Also for pipeline name I changed to a common variable, system.DefinitionName. But I did not find any common var for build and release Ids
![image](https://user-images.githubusercontent.com/12389221/55530324-4b183580-56c3-11e9-9f55-c1d762dfddc9.png)